samples/templates/default-azuredeploy.json

```diff
@@ -301,7 +301,7 @@
                     "apiVersion": "2018-11-01",
                     "location": "[resourceGroup().location]",
                     "properties": {
-                        "version": "3.1.11"
+                        "version": "5.0.0"
```

Comment:
Probably will need to change the extension above from `"AspNetCoreRuntime.3.1.x86"` too